README.md

```diff
@@ -1,18 +1,83 @@
 # apostrophe-salesforce-experience
-Not ready for use in any sense.
 
-Currently requires Node 8+.
+This module allows you to customize an ApostropheCMS website content experience for visitors based on their Salesforce profiles in a specific Salesforce instance. The developer will write [SOQL queries](https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/sforce_api_calls_soql.htm) to establish a set of "experiences," or Salesforce profiles. Website editors can then set specific widgets, pages, and pieces to display for a given set of those experiences.
```

Comment:
"It is important to note that "experiences" can map to anything at all in Salesforce. It depends entirely on the SOQL queries you choose to use."